src/components/showPokemon.js

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from "react"
+import Loading from 'react-simple-loading';
+
+export function ShowPokemon({Pokemon}){
+
+  const [pokemonData, setPokemonData] = useState([])
+  const [loading, setLoading] =useState(true)
+  const PokeURL = `https://pokeapi.co/api/v2/pokemon/${Pokemon}`
+
+  useEffect(()=>{
+    setLoading(true)
+    fetch(PokeURL)
+    .then(response => response.json())
```

Comment:
Why do you use callbacks (`.then/catch`) here and in `App.js` `async/await`?

Reply:
I wanted to use both possibilities, to see how they differ and how to reuse them